docs/infra/api-markdown-documenter/render-api-documentation.mjs

```diff
@@ -115,7 +143,6 @@ export async function renderApiDocumentation(inputDir, outputDir, uriRootDir, ap
 				}
 			},
 		},
-		newlineKind: "lf",
```

Comment:
Is this the default?

Reply:
This option was actually removed quite some time ago. It isn't relevant to the transformation stage, only the rendering stage (which this file does manually later).